Packet++/src/GtpLayer.cpp

```diff
@@ -155,522 +155,521 @@ GtpV1Layer::GtpExtension GtpV1Layer::GtpExtension::createGtpExtension(uint8_t* d
 
 GtpV1Layer::GtpV1Layer(GtpV1MessageType messageType, uint32_t teid)
 {
-    init(messageType, teid, false, 0, false, 0);
-
+	init(messageType, teid, false, 0, false, 0);
 }
 
 GtpV1Layer::GtpV1Layer(GtpV1MessageType messageType, uint32_t teid, bool setSeqNum, uint16_t seqNum, bool setNpduNum, uint8_t npduNum)
 {
-    init(messageType, teid, setSeqNum, seqNum, setNpduNum, npduNum);
+	init(messageType, teid, setSeqNum, seqNum, setNpduNum, npduNum);
 }
 
 void GtpV1Layer::init(GtpV1MessageType messageType, uint32_t teid, bool setSeqNum, uint16_t seqNum, bool setNpduNum, uint8_t npduNum)
 {
-    size_t dataLen = sizeof(gtpv1_header);
-    if (setSeqNum || setNpduNum)
-    {
-        dataLen += sizeof(gtpv1_header_extra);
-    }
+	size_t dataLen = sizeof(gtpv1_header);
+	if (setSeqNum || setNpduNum)
+	{
+		dataLen += sizeof(gtpv1_header_extra);
+	}
 
-   	m_DataLen = dataLen;
-	m_Data = new uint8_t[m_DataLen];
-	memset(m_Data, 0, m_DataLen);
+	m_DataLen = dataLen;
+	m_Data = new uint8_t[dataLen];
+	memset(m_Data, 0, dataLen);
```

Comment:
why does this change matter?

Reply:
All the same as for Packet's constructor before.